cosmos/job/drm/drm_ge.py

```diff
@@ -114,7 +114,7 @@ def qacct(task, timeout=600):
                 break
             except sp.CalledProcessError:
                 pass
-            time.sleep(1)
+            time.sleep(5)
```

Comment:
In general, or at least on the UGE installation I'm most familiar with, if `qacct` doesn't give a response, it probably won't a second later, either. If I understand UGE correctly, a primary reason why `qacct` would fail is because the qmaster daemon is down or failing over. Recovery takes more than a second in either case.